src/App.tsx

```diff
@@ -1,14 +1,16 @@
-/* eslint-disable max-len */
 import React from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { useTodoContext } from './hooks/useTodoContext';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
 export const App: React.FC = () => {
+  const { isLoading, showModal } = useTodoContext();
```

Comment:
you're using the context, that's great :+1:
it's not very necessary especially in this task because you can use simpler logic but looks great